cmd/minikube/cmd/docker-env.go

```diff
@@ -194,7 +194,8 @@ func mustRestartDockerd(name string, runner command.Runner) {
 	if err := sysinit.New(runner).Reload("docker"); err != nil {
 		klog.Warningf("will try to restart dockerd because reload failed: %v", err)
 		if err := sysinit.New(runner).Restart("docker"); err != nil {
-			exit.Message(reason.RuntimeRestart, `The Docker service within '{{.name}}' is not active`, out.V{"name": name})
+			klog.Warningf("Returning due to %v: The Docker service within '%v' is not active", reason.RuntimeRestart.ID, name)
```

Comment:
lets print out the error  for logs

klog.Warningf("couldn't restart docker inside minikbue because : %V)